cmd/netgoal/generate.go

```diff
@@ -472,3 +491,27 @@ func saveGenesisDataToDisk(genesisData gen.GenesisData, filename string) error {
 	}
 	return err
 }
+
+func generateAccountsLoadingFileTemplate(templateFilename, sourceWallet string, rounds, roundTrxCount, accountsCount, assetsCount, applicationCount uint64) error {
+
+	var data = remote.BootstrappedNetwork{
+		NumRounds:                 rounds,
+		RoundTransactionsCount:    roundTrxCount,
+		GeneratedAccountsCount:    accountsCount,
+		GeneratedAssetsCount:      assetsCount,
+		GeneratedApplicationCount: applicationCount,
+		SourceWalletName:          sourceWallet,
+	}
+	return saveLoadingFileDataToDisk(data, templateFilename)
+}
+
+func saveLoadingFileDataToDisk(data remote.BootstrappedNetwork, filename string) error {
+	f, err := os.OpenFile(filename, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0600)
+	if err == nil {
+		defer f.Close()
+
+		enc := codecs.NewFormattedJSONEncoder(f)
+		err = enc.Encode(data)
+	}
+	return err
```

Comment:
it is common to have a reversed check:

```suggestion
	if err != nil {
		return err
	}
	defer f.Close()
	enc := codecs.NewFormattedJSONEncoder(f)
	return enc.Encode(data)
```